spec/grid_spec.rb

```diff
@@ -2,75 +2,14 @@
 require_relative "../lib/grid.rb"
 
 RSpec.describe Grid do
-  describe "#coordinate_valid?" do
-    it "is true if in the first row and first column of the grid" do
-      valid_row_column = "A1"
+  describe "#mark" do
+    it "takes in a valid coordinate, and gives it a marked value in coordinates" do
+      coordinate = "A3"
       grid = Grid.new
 
-      expect(grid.coordinate_valid?(valid_row_column)).to eq true
-    end
-
-    it "is true if somewhere in the middle of the grid" do
```

Comment:
So, all of these tests are still important, since this is where the true logic resides.